modules/core/src/v2/coins/eth.ts

```diff
@@ -1268,6 +1269,11 @@ export class Eth extends BaseCoin {
         !_.isUndefined(buildParams.recipients) &&
         !_.isUndefined(buildParams.walletPassphrase)
       ) {
+        if (this instanceof Erc20Token) {
```

Comment:
Isn't there a method like `this.isToken` or something? `instanceof` always feels hacky

Reply:
I looked for `this.isToken` but didn't find it in BitGoJS. I believe that's just a platform thing. 
@tylerlevine do you know of a cleaner way of checking if I am a token?

Reply:
You could capture the 2nd constructor argument which is the coin from statics, and store as a property on `this`. That would have an `isToken` property you could check.

Reply:
Not sure which second constructor argument you're referring to... could you clarify?

Reply:
> Not sure which second constructor argument you're referring to... could you clarify?

Sure. When we instantiate a coin, it gets two objects passed to the constructor: the bitgo object and the statics coin (see https://github.com/BitGo/BitGoJS/blob/16427283f54d4183ce2acdda8e908b380fc1f693/modules/core/src/v2/coinFactory.ts#L71)

You're only capturing the first arg, but the second one has the info you need

Reply:
instantiating the coin is not really part of this PR, so maybe we leave that for another time and stick with `instanceof` here